pages/upgrade-guide/4.0.md

```diff
@@ -171,16 +219,31 @@ jQuery 4.0 adds support for [Trusted Types](https://developer.mozilla.org/en-US/
 
 This change was needed in order to add support for [Trusted Types](https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API). It's not entirely clear what real use cases might be affected by this change, but we're listing it here just in case.
 
-#### Important fix: Don't remove HTML comments from scripts
+#### Important fix: Don't remove HTML comments, including `CDATA` sections, from scripts
 
-Browsers can now handle this without jQuery's help. This change is not expected to affect existing code, but is an important fix because jQuery's logic removing those comments was not correctt in 100% of cases.
+Browsers can now handle this without jQuery's help. This change is not expected to affect existing code, but is an important fix because jQuery's logic removing those comments was not correct in 100% of cases.
```

Comment:
Since it's not just about comments:
```suggestion
Browsers can now handle these without jQuery's help. This change is not expected to affect existing code, but is an important fix because jQuery's logic removing them was not correct in 100% of cases.
```